dev-python/python-editor/python-editor-1.0.4-r1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( pypy3 python3_{7..9} )
+PYTHON_COMPAT=( pypy3 python3_{7..10} )
```

Comment:
How did you test it?

Reply:
Viewed the source code [here](https://github.com/fmoo/python-editor/blob/master/editor.py) (very simple therfore no problemt with 3.10)
and tried to run the test [here](https://github.com/fmoo/python-editor/blob/master/test.py).